modules/file_editor.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from modules.basemodule import BaseModule
+
+
+class FileEdit(BaseModule):
+    """
+    Use your mud client's builtin editor to modify files.
+
+    This is only really useful if pycat isn't running on the same machine as your client.
```

Comment:
May I ask, why is this useful at all? Isn't it more straightforward to SSH into the box with pycat, and use an actual editor like Vim or Emacs or Nano or something?

Reply:
The rest of the PR I agree with, by the way. Flattered that somebody's using it.

Reply:
I have it running in a docker container in a kubernetes cluster. The actual files are in a Persistent Volume, and are a lot harder to access than you'd think

Reply:
Besides, my mud client has a built in editor, and typing `#edit triggers.json` is always going to be more convenient than pulling up a terminal and doing stuff over ssh

Reply:
Right. Editing Docker volumes is a pain indeed.